src/Sylius/Component/Product/Model/VariantInterface.php

```diff
@@ -42,4 +42,14 @@ public function getAvailableOn();
      * @param null|\DateTime $availableOn
      */
     public function setAvailableOn(\DateTime $availableOn = null);
+
+    /**
+     * @return \DateTime
+     */
+    public function getAvailableUntil();
+
+    /**
+     * @param null|\DateTime $availableUntil
+     */
+    public function setAvailableUntil(\DateTime $availableUntil = null);
```

Comment:
I just followed the convention above. There was already setAvailableOn with default null value.